src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/LegacyCSProjPackageReferenceProject.cs

```diff
@@ -373,7 +373,7 @@ private static LibraryDependency ToPackageLibraryDependency(LegacyCSProjPackageR
             {
                 LibraryRange = new LibraryRange(
                     name: item.Name,
-                    versionRange: new VersionRange(new NuGetVersion(item.Version), originalString: item.Version),
+                    versionRange: VersionRange.Parse(item.Version),
```

Comment:
Parsing this as a range is correct. Should it also handle empty strings with a special message or catch errors here? What does the experience look like today?